### PR TITLE
Remove tool_conf.xml.main which is no longer used

### PR DESCRIPTION
**Pull request by nuwang**
Remove unused conf file.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [X] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Review by mvdbeek — APPROVED** [review]

**Comment by nuwang** [comment]
> There's still a reference in https://github.com/mvdbeek/galaxy/blob/ebf1e2435e52d05c109f8b6750924e915cf67ee5/run_tests.sh#L391, can you remove this too ?

Just did

File: run_tests.sh [hunk 1]
```diff
@@ -388,7 +388,7 @@ do
           shift 1
           ;;
       -main|-main_tools|--main_tools)
```
Comment (nsoranzo) [inline run_tests.sh:390]: Should this whole `main` case be removed from this script?

Comment (nuwang) [reply run_tests.sh:390]: @natefoo Any thoughts on this?

Comment (mvdbeek) [reply run_tests.sh:390]: I'd target the tool_conf.xml.sample with this instead (as you're doing now). We used to run jenkins tests against those tools (and we should probably also run those again in our workflows if we modify anything in `tools/`)

**Pull request merged**